src/Features/Core/Portable/SolutionCrawler/WorkCoordinator.SemanticChangeProcessor.cs

```diff
@@ -15,6 +15,8 @@ namespace Microsoft.CodeAnalysis.SolutionCrawler
 {
     internal sealed partial class SolutionCrawlerRegistrationService
     {
+        public const string EnqueueItem = nameof(EnqueueItem);
```

Comment:
❕ Needs a comment explaining what it's used for (specifically which actions use it as the tag)